src/universal/components/OutcomeCard/OutcomeCard.js

```diff
@@ -121,6 +122,9 @@ export default class OutcomeCard extends Component {
           }
         };
         cashay.mutate('updateTask', options);
+        if (autoFocus) {
+          dispatch(editingClearFocus());
```

Comment:
Our app has a single duck for controlling what requests focus from the UI. If focus was set on a card, it gets cleared on submit.